app/Http/Transformers/PostTransformer.php

```diff
@@ -23,6 +23,8 @@ public function transform(Post $post)
                 'url' => $post->url,
                 'caption' => $post->caption,
             ],
+            // @TODO: add the below line in when we use the transformer in the /tags endpoint.
+            // 'tagged' => $post->tagNames(),
```

Comment:
oh good call, @katiecrane ! I just uncommented it and reviewed a post and nothing broke! Do you think I should just add this now?